Ginger/GingerAutoPilot/APIModelLib/SwaggerApi/OpenApiVer3.cs

```diff
@@ -80,6 +83,7 @@ public ObservableList<ApplicationAPIModel> OpenApiThree(SwaggerDocument Swaggerd
                                         if (Operation.RequestBody != null)
                                         {
                                             AAM.AppModelParameters.Append(GenerateXMLBody(AAM, Operation.RequestBody.Content.ElementAt(0).Value.Schema));
+                                            SetOptionalValue(AAM.AppModelParameters, listExampleValues);
```

Comment:
Please move this function call outside of switch so this function call will be generic for all cases